src/xFactor/Judge.java

```diff
@@ -0,0 +1,41 @@
+package xFactor;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class Judge extends Person {
+	private int maxFavourites;
+	private boolean isSpecial;
+	
+	public Judge(String fName, String lName, int age, Gender male, String birthPlace, int max) {
+		super(fName, lName, age, male, birthPlace);
+		Competition.numberOfJudges++;
+		Competition.getJudges().add(this);
+		this.maxFavourites = max;
+	}
+	
+	public int getMaxFavourites() {
+		return maxFavourites;
+	}
+	
+	public boolean isSpecial() {
+		return isSpecial;
+	}
+	
+	public void setSpecial(boolean isSpecial) {
+		this.isSpecial = isSpecial;
+	}
+	
+	public void addFavourite(Participant participant, Stage stage) {
+		Set<Participant> participants = new HashSet<Participant>();

```

Comment:
You can have Map<StageId, List<Participant>> and map judge -> favorites like that, do not manipulate Stage in another POJO.
Think of a method like something that does exact one thing. You either add favorites to judge or to stage, not both.

Reply:
do you need this at all?